deployer/deployer-services/src/main/java/io/hyscale/deployer/events/model/AwaitingDeploymentEvent.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.deployer.events.model;
+
+import io.hyscale.commons.framework.events.model.ActivityEvent;
+import io.hyscale.commons.framework.events.model.ActivityState;
+import io.hyscale.commons.models.ServiceMetadata;
+
+public class AwaitingDeploymentEvent extends ActivityEvent {
```

Comment:
You are firing an InProgress Event which is irrelevant. Fire a Deployment Completion Event it helps users to perform some action upon deployment completion